### DISCUSSION TITLE
I just want to move the floater arrow position

### DISCUSSION THREAD

**Discussion by ashish3010** [gilbarbara/react-joyride#878] [Q&A]
![IMG_20221228_115511](https://user-images.githubusercontent.com/69194675/209768061-1abf6919-b730-4bd5-9a7f-d57f994a75f7.jpg)
Please tell me how to move the floater arrow position.
Because by giving the border radius to the tooltip container, arrow style is distorted, and I want to give marginLeft or marginRight to the floater arrow.
Or can I use custom arrows for specific tooltip container ?
Please help

**Comment by gilbarbara** [comment]
Hey @ashish3010

Please provide a minimal reproducible example in https://codesandbox.io/ or similar.

**Comment by bastrabino** [comment]
@ashish3010 can you tell me how you solved please?
